pkg/cmd/cli/cmd/logs.go

```diff
@@ -63,6 +69,7 @@ func NewCmdLogs(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Com
 		}
 		cmdutil.CheckErr(o.RunLog())
 	}
+	cmd.Flags().String("version", "", "View the logs of the deployment associated with the provided version for a deployment config")
```

Comment:
> Since you're here and builds also have numbers, can you wire that up as well?

Currently in deployments the object gets resolved in the server while in builds we do this in the client. What you are asking for can only be achieved by adding a version field in `BuildLogOptions` and doing the same that already happens in deployments (or we could continue to resolve the object in the client but that would still be incosistent). Is this fine?